Makefile

```diff
@@ -4,13 +4,11 @@ PREFIX ?= /
 
 .PHONY: install
 install:
-	install -D -m0644 src/overdue.awk $(PREFIX)/usr/share/overdue/overdue.awk
 	install -D -m0644 src/overdue-libraries-upgrade.hook \
 		$(PREFIX)/usr/share/libalpm/hooks/overdue-libraries-upgrade.hook
-	install -D -m0755 src/overdue.sh  $(PREFIX)/usr/bin/overdue
```

Comment:
I create a script that pipes all library files into the script and it seems almost instant now with the most recent changes.